CRM/Upgrade/Incremental/php/FiveTwentyThree.php

```diff
@@ -167,4 +170,20 @@ private function hasConfigBackendData() {
     && CRM_Core_DAO::singleValueQuery('SELECT count(*) c FROM `civicrm_domain` WHERE config_backend IS NOT NULL') > 0;
   }
 
+  /**
+   * Install sequentialCreditNotes extension.
+   *
+   * This extension is being moved from core functionality to an extension.
+   *
+   * @param \CRM_Queue_TaskContext $ctx
+   *
+   * @return bool
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  public static function installCreditNotes(CRM_Queue_TaskContext $ctx) {
+    civicrm_api3('Extension', 'install', ['sequentialcreditnotes']);
```

Comment:
So if one is accepting the premise/goal and focused on technical aspects of implementation - would it be fair to say that this task (*automatically enabling the extension during an upgrade*) is perhaps the main "new" thing going on? e.g. for reviewing, it's important to do some `r-run` on this aspect?

Reply:
I separated it out https://github.com/civicrm/civicrm-core/pull/16513